src/net/net.c

```diff
@@ -87,7 +88,7 @@ void *lwip_allocate(u64 size)
 {
     /* To maintain the malloc/free interface with mcache, allocations must stay
        within the range of objcaches and not fall back to parent allocs. */
-    assert(size <= U64_FROM_BIT(MAX_LWIP_ALLOC_ORDER));
+    assert(size <= U64_FROM_BIT(lwip_alloc_order));
```

Comment:
I think we can remove this assert() now, since it would make writes to network socket file descriptors unable to handle data sizes larger than 2KB in low-memory VMs (e.g. the netsock and tun runtime tests would fail in such VMs); so we don't even need the lwip_alloc_order global variable.
Likewise, the check against the max alloc order in mbedtls_calloc() in klibs/mbedtls.c could be removed.

Reply:
Removed, thanks!

Reply:
The lwip_alloc_order global variable is no longer needed and can be removed.
Also, the check against the max alloc order in mbedtls_calloc() in klibs/mbedtls.c can be removed.

Reply:
Ok, update pushed.